File: date_dow/java_date_dow/DateDow.java
import java.util.Calendar;

public class DateDow {
  public static void main( String[] args ) {
    if( args.length != 0 ) {
      System.err.println( "Usage: java -jar DateDow.jar <noargs>");
      System.exit(1);
    }
    Calendar now = Calendar.getInstance();
    System.out.println( "Current date : " + (now.get( Calendar.MONTH ) + 1) + "-"
                        + now.get( Calendar.DATE ) + "-" + now.get( Calendar.YEAR ));

    String[] strDays = new String[] { "Sunday", "Monday", "Tuesday", "Wednesday", "Thusday",
                                      "Friday", "Saturday" };
    // Day_OF_WEEK starts from 1 while array index starts from 0
    System.out.println( "Current day is : " + strDays[now.get( Calendar.DAY_OF_WEEK ) - 1] );
  }
}

File: date_dow/python_date_dow/date_dow.py
import sys
import os
import time
#from datetime import date
import datetime

def date_dow():
  dayofWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
  x = datetime.datetime.today().weekday()
  return dayofWeek[x]
  #time.time()
  #oldDate = date(2010, 02, 06)  # year, month, day
  #dayofWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
  #print ("The day of the week on %s was a %s" % (oldDate.strftime("%d %b %Y"), dayofWeek[date.weekday(oldDate)]))
  #1226392970.4683549
  #time.gmtime(time.time()).tm_mday
  #time.gmtime(time.time()).tm_mon
  #time.gmtime(time.time()).tm_year
  #x = time.gmtime(time.time())

def main():
  if len(sys.argv) != 1:
    print( "Usage: %s <noargs>" % sys.argv[0])
    sys.exit()
  print(date_dow())
  sys.exit(), end

main()



